apps/server/src/shared/domain/interface/system-provisioning.strategy.ts

```diff
@@ -1,5 +1,5 @@
 export enum SystemProvisioningStrategy {
-	SCHULCONNEX_LEGACY = 'sanis',
+	SCHULCONNEX_LEGACY = 'moin.schule',
```

Comment:
This must not be changed. This is the old provisioning strategy. (which we should remove at some point...)